CodeStarterCamp_Week5/main.swift

```diff
@@ -18,6 +18,13 @@ let chun = TalentedPerson(name: "chun", height: 176, singing: .B, dancing: .B, a
 let ame = TalentedPersonWithBadPersonality(name: "ame", height: 155, singing: .C, dancing: .A, acting: .A, frequancyOfCursing: .B)
 let groot = TalentedPersonWithBadPersonality(name: "groot", height: 200, singing: .A, dancing: .A, acting: .C, frequancyOfCursing: .C)
 
-var yagom = AuditionManager(totalApplicantsList: [snoopdog, rowan, wongbing, jun, chun, ame, groot])
+var auditionManager = AuditionManager(applicants: [snoopdog, rowan, wongbing, jun, chun, ame, groot])
```

Comment:
```suggestion
let applicants = [snoopdog, rowan, wongbing, jun, chun, ame, groot]
var auditionManager = AuditionManager(applicants: applicants)
```

이렇게 파라미터에 데이터를 직접 작성하는 모습보단 한번 상수에 담아서 처리해주는 것이 더 깔끔할 것 같습니다